### ISSUE TITLE
SSH Guidelines : Consider adding sntrup key exchange method

### ISSUE THREAD

**Issue by Fernandokitten** [mozilla/infosec.mozilla.org#170]
As of version 8.0, OpenSSH supports a post-quantum key exchange using the sntrup4591761x25519-sha512@tinyssh.org key exchange.

In OpenSSH 8.5, the previous sntrup4591761x25519-sha512@tinyssh.org method is replaced with sntrup761x25519-sha512@openssh.com.

**Comment by hexkey** [comment]
That's correct, it's mentioned in https://www.openssh.com/releasenotes.html

>   The previous sntrup4591761x25519-sha512@tinyssh.org method is
>   replaced with sntrup761x25519-sha512@openssh.com. Per its
>   designers, the sntrup4591761 algorithm was superseded almost two
>   years ago by sntrup761.

**Comment by landryb** [comment]
This is even more true that the page mentions openssh 6.7 as 'modern' version (released in 2014..) and openssh 10.1 will show a loud warning if you dont have a PQ algorithm, cf https://www.openssh.com/pq.html

so .. can someone please add modern algorithms ?
- [mlkem768x25519-sha256](https://datatracker.ietf.org/doc/draft-ietf-sshm-mlkem-hybrid-kex/)  is supported since 9.9 (ok, 2025..)
- [sntrup761x25519-sha512](https://datatracker.ietf.org/doc/draft-josefsson-ntruprime-ssh/) is supported since 9.0 (2022)

but sadly, since i see #176 and #192 are unreviewed.. it'll stay this way ?
